Knossos.NET/Views/DebugView.axaml

```diff
@@ -10,18 +10,21 @@
 	<Grid RowDefinitions="Auto,*">
 		<StackPanel>
 			<WrapPanel Grid.Row="0" Margin="5">
-				<Button Margin="5" Classes="Primary" Command="{Binding OpenFS2Log}">View fs2__open.log</Button>
-				<Button Margin="5" Classes="Primary" Command="{Binding UploadFS2Log}">Upload fs2__open.log</Button>
-				<Button Margin="5" Classes="Settings" Command="{Binding OpenDebugFilterView}">Adjust Debug Filters</Button>
-				<Button Margin="5" Classes="Settings" Command="{Binding OpenFS2Ini}">View fs2__open.ini</Button>
+				<Button Margin="5" Width="175" Classes="Primary" Command="{Binding UploadFS2Log}">Upload fs2__open.log</Button>
+				<Button Margin="5" Width="175" Classes="Primary" Command="{Binding OpenFS2Log}" ToolTip.Tip="View the logfile output from Freespace Open's Debug Build">Open fs2__open.log</Button>
+				<Button Margin="5" Width="175" Classes="Settings" Command="{Binding OpenDebugFilterView}">Adjust Debug Filters</Button>
+				<Button Margin="5" Width="175" Classes="Settings" Command="{Binding OpenFS2Ini}">View fs2__open.ini</Button>
 			</WrapPanel>
 			<WrapPanel Grid.Row="0" Margin="5">
-				<Button Margin="5" Classes="Settings" Command="{Binding UploadKnossosConsole}">Upload KnossosNET Console</Button>
-				<Button Margin="5" Classes="Settings" Command="{Binding OpenLog}">View KnossosNET Logfile</Button>
-				<Button Margin="5" Classes="Settings" Command="{Binding OpenSettings}">View KnossosNET Settings File</Button>
+				<Button Margin="5" Width="175" Classes="Primary" Command="{Binding UploadKnossosConsole}">Upload Knossos Log</Button>
```

Comment:
Current protocol has been to use `KnossosNET` as all public facing mentions of Knossos, so my only suggestion is the use that in these cases :)

Reply:
I will implement this, but it's looking like it makes my current layout too wonky.  I am going to rearrange a couple things.